docs/enterprise-guide/enterprise-cloud-guide/faq/README.md

```diff
@@ -47,7 +47,7 @@ To import your community instance into our cloud environment, please send us the
 
 For a package-based installation, you can create both as root user on your environment as follows: `openproject run backup`
 This creates the attachment and postgresql-dump or mysql-dump under /var/db/openproject/backup.
-If you are still running OpenProject under MySQL, your dump will be converted to PostgreSQL before importing, we will do this for you. More information about the backup tool can be found [here](../../operations/backup/backup-guide-packaged-installation/).
+If you are still running OpenProject under MySQL, your dump will be converted to PostgreSQL before importing, we will do this for you. More information about the backup tool can be found [here](../../../operations/backup/backup-guide-packaged-installation/).
 Before uploading the attachments securely to us using the [this form](https://openproject.org/saas-import), please contact us via support@openproject.com.
```

Comment:
We discussed with Markus that we can retire the form to upload data. Instead, users should attach their SQL dump and attachment folder to a work package in their OpenProject (Cloud) environment and send us the link to that work package. We can then import their data from there.
(See also [this template](https://app.hubspot.com/templates/5842801/edit/11124411).)